tests/io/test.xpi.js

```diff
@@ -542,19 +532,17 @@ describe('Xpi.getFileAsStream()', function() {
         expect(err.message).toContain('¡hola!');
       });
   });
-
 });
 
-describe('Xpi.getFilesByExt()', function() {
-
+describe('Xpi.getFilesByExt()', function getFilesByExtCallback() {
```

Comment:
Why are they named now? Can't we just use lambdas here as well, afair we do that quite often in other tests?